deps/v8/include/libplatform/v8-tracing.h

```diff
@@ -59,11 +59,19 @@ class V8_PLATFORM_EXPORT TraceObject {
       const char** arg_names, const uint8_t* arg_types,
       const uint64_t* arg_values,
       std::unique_ptr<v8::ConvertableToTraceFormat>* arg_convertables,
+#if defined(V8_OS_ZOS)
+      unsigned int flags, int pid, pthread_t tid, int64_t ts, int64_t tts,
+#else
       unsigned int flags, int pid, int tid, int64_t ts, int64_t tts,
+#endif
       uint64_t duration, uint64_t cpu_duration);
 
   int pid() const { return pid_; }
+#if defined(V8_OS_ZOS)
+  pthread_t tid() const { return tid_; }
```

Comment:
Is tid() only used for display?  If so, should we return int(tid.__ & 0x7fffffff) here rather than modify the return result below

Reply:
Yes it seems tid() is used only display. But I don't think we should tid() since it would then lose its definition.